plugins/foreman_discovery/8.0/index.md

```diff
@@ -0,0 +1,1302 @@
+---
+layout: plugin
+pluginname: foreman_discovery
+title: Foreman Discovery 8.0 Manual
+version: 8.0
+# versions for matrix and snippets
+# (use short version for imgver e.g. 3.0)
+pluginver: 8.0.0
+proxyver: 1.0.3
+imgver: 3.3
+cliver: 0.0.3
+# uncomment to show warning box for an old release
+#warning: old
```

Comment:
Yeah I was thinking 1.14 is not yet out but this is irrelevant. Will change this and the others thanks.